src/network/networkprotocol.h

```diff
@@ -173,7 +178,12 @@ enum ToClientCommand
 		NOTE: The position in here is deprecated; position is
 		      explicitly sent afterwards
 	*/
-
+	TOCLIENT_ACCESS_DENIED = 0x0A,
+	/*
+		u16 command
```

Comment:
remove u16 command, we don't care about u16 command with the NetworkPacket design, it's implicit :)